.gitignore

```diff
@@ -83,7 +83,6 @@ tools/tests/reports/testdox.html
 tools/tests/reports/testdox.txt
 tools/extensions/org.civicrm.contactlayout
 tools/extensions/phpstorm/.phpstorm.meta.php
-l10n
```

Comment:
Suggestion: `/l10n`

That will allow it to ignore top-level `./l10n` folder while committing nested folders like `./settings/l10n`.